app/views/editor/posts/index.html.haml

```diff
@@ -22,7 +22,10 @@
       %tr{ class: post.published? ? nil : 'post--unpublished' }
         %td= post.public_id
         %td= post.title
-        %td= post.category&.name
+        %td
+          %ul
+            - post.categories.pluck(:name).each do |name|
```

Comment:
`AR::Relation#pluck` fires SQL query. 
If there are no reason for using `pluck`, can you use eager loading?